src/Notifications/AbstractKillmailNotification.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of SeAT
+ *
+ * Copyright (C) 2015 to 2021 Leon Jacobs
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+namespace Seat\Notifications\Notifications;
+
+use AbstractNotification;
+
+/**
+ * Class AbstractKillmailNotification.
+ *
+ * @package Seat\Notifications\Notifications
+ */
+class AbstractKillmailNotification extends AbstractNotification
+{
+    /**
+     * Get the Slack representation of the notification.
+     *
+     * @param $notifiable
+     * @return bool
+     */
+    protected function shouldProcessKillmail($notifiable) {
+
+        $killmailConfig = config('notifications.killmails');
```

Comment:
I think we should keep config for either static content or server configuration (ie: mysql, redis, menu etc...)

What about using `settings` instead ? We would create a new pane in `SeAT Settings` area where instance owner would be able to define corporations and alliances to which the instance is related.
Other features like limited sign-in could also benefits of such settings.